Cloud integration update

robalty opened the pull request:
Added static files for gcloud deployment, plus app.yaml and main.py for the same. Edited settings.py to point to the Gcloud postgresql instance.
I'll leave the app alive online for now, although having the API just publicly available isn't ideal. I'll do some research into how we can make that more secure.

robalty committed:
Added static files for gcloud deployment, plus app.yaml and main.py for the same. Edited settings.py to point to the Gcloud postgresql instance.

SimonBartonPSU commented:
Awesome work, although I don't think I follow what's going on entirely. I get changing the settings to point to a postgres instance.

But how did the static files get generated?

And how does this get deployed? `gsutil` command that references GitHub? Might help once I can see the gcloud project

robalty commented:
@SimonBartonPSU The static files are actually generated by manage.py - it's just making fixed pathways to files that gcloud needs. It's just bookkeeping for the gcloud instance. Deployment is done with the Gcloud SDK - from the folder with app.yaml you just run "gcloud app deploy" to deploy. Since deployment is already done though, and it's online already, you can just run the cloud sql proxy and run manage.py locally.

SimonBartonPSU commented:
Groovy roovy, these seems even easier than the app engine

robalty committed:
Removed static files, since those can be generated at deployment time

robalty committed:
Update .gitignore

robalty committed:
Add files via upload

seansisson committed:
Add react-native prefix to some ignores

seansisson committed:
Change Post to Posting in admin.py

robalty committed:
Add files via upload

SimonBartonPSU commented on django/todoproj/settings.py:
```diff
@@ -25,84 +25,105 @@
+            'ENGINE': 'django.db.backends.postgresql',
+            'HOST': '/cloudsql/cellular-virtue-277000:us-central1:capstone-covid',
+            'USER': 'postgres',
+            'PASSWORD': 'testpassword',
```
we may want to find a way to conceal our password and not publish it

unless this is a placeholder in which case I have derped

SimonBartonPSU committed:
Add cloudbuild file for continuous deployment

Related to #31
Created a Cloud Source Repository on GCP mirroring our repo
Set up a cloud build trigger that will run this yaml, which runs
"gcloud app deploy" when a push to master occurs
The cloud build trigger is currently disabled

actually add the yaml text
